src/cdpTargetsProvider.ts

```diff
@@ -78,6 +78,9 @@ export default class CDPTargetsProvider implements vscode.TreeDataProvider<CDPTa
                     return a.targetJson.type < b.targetJson.type ? -1 : 1;
                 }
             });
+            // Raise and quickly hide a notification so the screen reader announces the new state of the targets list
+            vscode.window.showInformationMessage(`Target list refreshed. ${targets.length} ${targets.length === 1 ? "target": "targets"} found.`);
+            setTimeout(() => vscode.commands.executeCommand("notifications.hideToasts"), 10)

```

Comment:
I think we should remove the `notifications.hideToasts` call unless we can specify our refresh toast.  We don't want to unexpectedly remove any other toast notifications for the user.

Reply:
Ok, I'll do that.
But I looked into this a little more and found a way to make only our toast close after a timeout using `vscode.window.withProgress`. I think makes a better UX for sighted users, the only thing is that NVDA doesn't announce the message, but Narrator does. With `showInformationMessage`, both NVDA and Narrator announce the message, but the toasts can stack up and the notification list can quickly get flooded with refresh messages. Which one should we go with?

Reply:
I took a look at the discussion at:
https://github.com/microsoft/vscode/issues/114718

and IMO I think is better to wait for those fixes to land. (We could +1 the issue by mentioning it on our bug to improve visibility). I see value on this solution but I'm not sure it will fix the issue but just temporarily mitigate it, thoughts?

Reply:
I agree, I'll keep an eye on that discussion and follow up with this once the fixes land in vscode. Hopefully before July when all the accessibility bugs need to be fixed!